argh/tests/lib.rs

```diff
@@ -998,6 +1044,7 @@ fn redact_arg_values_positional() {
     #[derive(FromArgs, Debug)]
     /// Short description
     struct Cmd {
+        #[allow(dead_code)]
```

Comment:
Maybe there is a clue here that, perhaps, the new warnings with `argh` marked fields is different from the issues resulting from rust-lang/rust#85200 (discussed in my other comment below).

I _thought_ `#[allow(unused)]` would work. (Just looking at some historical conversations and how we worked around the sudden appearance of new warnings about unused fields in structs with `derive`d traits.) I realize this is an example, but would it be better to specify this override as `unused` instead of `dead_code`?

Reply:
`#[allow(unused)]` works, thanks